### ISSUE TITLE
Ability to run projects simultaneously in warrior

### ISSUE THREAD

**Issue by chfoo** [ArchiveTeam/seesaw-kit#50]
Related to #37 

https://twitter.com/mrtsoft/status/414750530704269312: 

<blockquote>
suggestion 2: let a user run multiple projects inside warrior as some projects like yahoo return 999 ;
</blockquote>

**Comment by endolith** [comment]
Yeah this seems really wasteful of our resources.  I went through the trouble of setting up a warrior, selected "ArchiveTeam’s Choice" and ... it's just sleeping.

> Starting CheckIP for Item 
> Finished CheckIP for Item 
> Starting GetItemFromTracker for Item 
> Received item 'group-meta:6069883' from tracker
> Starting PrepareDirectories for Item group-meta:6069883
> Finished PrepareDirectories for Item group-meta:6069883
> Starting WgetDownload for Item group-meta:6069883
> Archiving item group-meta:6069883
> 1=200 https://groups.roblox.com/v1/groups/6069883 
> 2=200 https://www.roblox.com/communities/6069883/Its-ap 
> 3=429 https://groups.roblox.com/v2/groups/6069883/wall/posts?limit=100&sortOrder=Asc 
> Server returned bad response (RETRFINISHED). Sleeping 703 seconds.

Shouldn't it switch to another project while it waits?
